flight/boundary/dropoff.py

```diff
@@ -0,0 +1,44 @@
+import plotter
+import point_finder
+
+
+flyZones = [
+    {
+        "altitudeMin": 75.0,
+        "altitudeMax": 400.0,
+        "boundaryPoints": [
+            {"latitude": 38.31442311312976, "longitude": -76.54522971451763},
+            {"latitude": 38.31421041772561, "longitude": -76.54400246436776},
+            {"latitude": 38.3144070396263, "longitude": -76.54394394383165},
+            {"latitude": 38.31461622313521, "longitude": -76.54516993186949},
+            {"latitude": 38.31442311312976, "longitude": -76.54522971451763},
+        ],
+    }
+][0]
```

Comment:
Why are you assigning this as a list and then immediately indexing the first element before discarding it? Couldn't you just assign `flyZones` to the element?